### ISSUE TITLE
What is the correct way to troubleshoot when automatic cert renewal isn't happening?

### ISSUE THREAD

**Issue by marcchampoux** [sjkp/letsencrypt-siteextension#384]
Hi!

First and foremost: thank you for this wonderful extension... it has allowed me to quickly and easily ssl enable a lot of sites running on Azure.

The problem that I have is that automatic certificate renewal does not seem to be happening... at all.

I have read the wiki, including the entries on troubleshooting and I did some research in the "issues" tab and tried a few things... but automatic cert renewal isn't happening...

In case you are wondering: I just tried to request a new cert and it worked very nicely (again) so I guess the app is installed/configured correctly.

The Webjob is in the "Running" state and I can't find any issues with it...

Any tips and tricks?

Thank you!

Marc

**Comment by joelnotified** [comment]
Hi @marcchampoux 
Currently experiencing the same problem. Cert renewal has stopped working for us. Have you found a solution?

**Comment by marcchampoux** [comment]
Hi Joel... I have not found a solution. I'm hoping someone else will be able to give us some points on what to look for to figure out why the certs are not renewing.

**Comment by joelnotified** [comment]
@marcchampoux Looks like it could be related to this in my case, so I'm hoping there will be a merge and release of that pretty soon: https://github.com/sjkp/letsencrypt-siteextension/pull/382

**Comment by marcchampoux** [comment]
Hi Joel,

Thanks for letting me know about the merge request... it was done today and I updated the extension from 1.0.5 to 1.0.6 as soon as I saw the merge.

After restarting the web app, I monitored the web job log and it worked!

The web job started to renew the certs!

The webjob did run into a few problems when it tried to renew some of the certs: it turns out someone had pointed some domain names to other servers and hadn't told me... so after doing a bit of cleanup, it all worked nicely...

Now I just need to wait until the next renewal, I think in 68 days (the default is 90 days minus 22 right?) to see if all the certs get automatically renewed...

Thanks again!

Marc
